### ISSUE TITLE
Cannot print the help for any command

### ISSUE THREAD

**Issue by mollyIV** [tuist/tuist#1564]
**Describe the bug**
After the upgrade to the newest `tuist` version ([1.12.0](https://github.com/tuist/tuist/releases/tag/1.12.0) in a time of writing) via `tuist update` command, I cannot print the help for any command. 

**To Reproduce**

1. Update `tuist` to the newest version (1.12.0) via `tuist update` command in your terminal.
2. Type `tuist --help` command in your terminal. 
3. Notice that the help cannot be printed out and there's an error instead. 

**Expected behavior**
A help of a command is printed out. 

**Logs**
```bash
$ tuist update
Checking for updates...
There are no updates available
$ tuist version
1.7.1
$ tuist --help
We received an error that we couldn't handle:
    - Localized description: The operation couldn’t be completed. (ArgumentParser.CommandError error 1.)
    - Error: CommandError(commandStack: [TuistKit.TuistCommand], parserError: ArgumentParser.ParserError.helpRequested)"
```

**Additional info**
What's interesting is that even though there's no updates available for `tuist`, the `version` command shows `1.7.1` instead of `1.12.0`.

**Desktop (please complete the following information):**

- OS: macOS 10.15.5
- Swift: 5.2.4 
- Command Line Tools: Xcode 11.5

Please let me know if you need more info 🙇

**Comment by pepibumur** [comment]
Hey @mollyIV, thanks for bringing this up. Based on your description, it seems that the issue you are running into is [this one](https://github.com/tuist/tuist/pull/1562) that @kwridan just fixed. I'll release a hotfix release later today to include this fix. Would you mind trying it out when it's ready and closing this issue if that fixes the problem you are facing?

**Comment by pepibumur** [comment]
We've released Tuist 1.12.1. Would you mind running `tuist update` and trying again @mollyIV?

**Comment by mollyIV** [comment]
Hey @pepibumur 👋 

Sure! I did the testing and I confirm that I am able to print out the help 🎊 

```bash
❯ ~ tuist update
Checking for updates...
Installing new version available 1.12.1
Downloading version 1.12.1
Installing...
Version 1.12.1 installed
❯ ~ tuist version
1.12.1
❯ ~ tuist --help
OVERVIEW: Generate, build and test your Xcode projects.

USAGE: tuist <subcommand>

OPTIONS:
  -h, --help              Show help information.

SUBCOMMANDS:
  generate                Generates an Xcode workspace to start working on the
                          project.
  up                      Configures the environment for the project.
  focus                   Opens Xcode ready to focus on the project in the
                          current directory
  edit                    Generates a temporary project to edit the project in
                          the current directory
  secret                  Generates a cryptographically secure secret.
  dump                    Outputs the project manifest as a JSON
  graph                   Generates a dot graph from the workspace or project
                          in the current directory
  lint                    Lints a workspace or a project that check whether
                          they are well configured
  version                 Outputs the current version of tuist
  build                   Builds a project
  create-issue            Opens the GitHub page to create a new issue
  scaffold                Generates new project based on template
  init                    Bootstraps a project
  cloud                   A set of commands for cloud-related operations
  signing                 A set of commands for signing-related operations
  clean                   Clean all the artefacts stored locally

  See 'tuist help <subcommand>' for detailed help.
```

Thanks a lot for such a quick fix 💪
